client/my-sites/plans-v2/types.ts

```diff
@@ -130,3 +132,15 @@ export interface AvailableProductData {
 	product_slug: string;
 	cost: number;
 }
+
+export interface FilterBarProps {
```

Comment:
For readability, I'd day the component props should stay in the component file unless they're imported elsewhere.